src/sharedServices/dynamoDB/dynamoDBUpdate.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  UpdateItemCommand,
+  UpdateItemCommandInput
+} from '@aws-sdk/client-dynamodb'
+import { DynamoDBParams } from '../../types/dynamoDBParams'
+import { isDataRequestParams } from '../../types/dataRequestParams'
+import { getEnv } from '../../utils/helpers'
+import { ddbClient } from './dynamoDBClient'
+
+export const updateQueryByZendeskId = async (
+  zendeskId: string,
+  attributeKey: string,
+  attributeValue: string
+): Promise<DynamoDBParams> => {
+  const params: UpdateItemCommandInput = {
+    TableName: getEnv('DYNAMODB_TABLE_NAME'),
+    Key: { zendeskId: { S: zendeskId } },
+    ReturnValues: 'ALL_NEW',
```

Comment:
NOTE - at present the update call returns all of the entry in the DB, including the updated fields. It is possible to configure it so that only the updated attributes are returned.

Reply:
I think it's fine to return the whole entry 👍🏻